src/main/java/org/jabref/gui/maintable/MainTable.java

```diff
@@ -123,6 +134,35 @@ public MainTable(MainTableDataModel model, JabRefFrame frame,
         database.getDatabase().registerListener(this);
     }
 
+    /**
+     * This is called, if a user starts typing some characters into the keyboard with focus on main table.
+     * The tableview will scroll to the cell with the same starting column value and typed string
+     * @param sortedColumn The ListView currently focused
+     * @param keyEvent The pressed character
+     */
+
+    private void jumpToSearchKey(TableColumn<BibEntryTableViewModel, ?> sortedColumn, KeyEvent keyEvent) {
+        if (keyEvent.getCharacter() == null || sortedColumn == null) {
+            return;
+        }
+
+        if (System.currentTimeMillis() - lastKeyPressTime < 700) {
+            columnSearchTerm += keyEvent.getCharacter().toLowerCase();
+        } else {
+            columnSearchTerm = keyEvent.getCharacter().toLowerCase();
+        }
+
+        lastKeyPressTime = System.currentTimeMillis();
+
+        this.getItems().stream()
+            .filter(item -> Optional.ofNullable(sortedColumn.getCellObservableValue(item).getValue())
+                                    .map(Object::toString)
+                                    .orElse("")
+                                    .toLowerCase().startsWith(columnSearchTerm))
+            .findFirst()
+            .ifPresent(item -> { this.scrollTo(item); this.clearAndSelect(item.getEntry()); });
```

Comment:
You can put these statements inside the lambda expression in multiple lines.
Sorry, I think my previous comment was a bit misunderstandable.

Besides that, it looks good to me. let's see what a second developer says.

Reply:
Sorry, I don't quite understand what your mean. You mean I should shrink the code in the lambda expression into one or two lines instead of occupying multiple lines?

Reply:
```suggestion
            .ifPresent(item -> {
                this.scrollTo(item);
                this.clearAndSelect(item.getEntry());
            });
```